core/object/callable_method_pointer.h

```diff
@@ -37,7 +37,7 @@
 #include <type_traits>
 
 class CallableCustomMethodPointerBase : public CallableCustom {
-	uint32_t *comp_ptr = nullptr;
+	const char *comp_ptr = nullptr;
```

Comment:
I think i'd prefer `uint8_t*` for the pointer. `char` is more often associated with text.

Reply:
Done. I originally did this just to match the language spec "[a char, unsigned char, or std​::​byte type](https://eel.is/c++draft/basic.lval#11.sentence-1)", but it's not realistically going to be a problem.